From a7d0ce1deda0699343099a02c25fe55dc98c010d Mon Sep 17 00:00:00 2001
From: Evgeni Golov <evgeni@debian.org>
Date: Thu, 8 Oct 2015 08:50:26 -0700
Subject: Do not build issues if DEB_BUILD_ISSUES=disabled

   Building the issues via sphinxcontrib.issuetracker requires
   a network connection which is not available on Debian
   buildds.
Bug-Debian: http://bugs.debian.org/681379
Patch-Name: no-issues.patch
---
 docs/conf.py | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index fad2a07..7d12486 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -81,6 +81,7 @@ html_sidebars = {
 }
 
 # ## Issuetracker
-issuetracker = 'github'
-issuetracker_project = 'celery/django-celery'
-issuetracker_issue_pattern = r'[Ii]ssue #(\d+)'
+if os.environ.get("DEB_BUILD_ISSUES", False) != "disabled":
+    issuetracker = 'github'
+    issuetracker_project = 'celery/django-celery'
+    issuetracker_issue_pattern = r'[Ii]ssue #(\d+)'
